docs/DEVELOPING.md

```diff
@@ -343,6 +343,111 @@ $ make ftest NAME=azure_blob_storage
 
 ℹ️ The e2e test uses default values defined in [configure Azure Blob Storage connector](#configure-azure-blob-storage-connector)
 
+
+## PostgreSQL Connector
+
+The [Elastic PostgreSQL connector](https://github.com/elastic/connectors-python/blob/8.7/connectors/sources/postgresql.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **beta** starting in 8.7_.
+Features in beta are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
+### Setup and basic usage
+
+Complete the following steps to deploy the connector:
+
+1. [Gather PostgreSQL details](#gather-postgresql-details)
+2. [Configure PostgreSQL connector](#configure-postgresql-connector)
+
+#### Gather PostgreSQL details
+
+Collect the information that is required to connect to your PostgreSQL server:
+
+- The server host address where the PostgreSQL is hosted.
+- The port where the PostgreSQL is hosted.
+- The username the connector will use to log in to the PostgreSQL.
+- The password the connector will use to log in to the PostgreSQL.
+- The database name where the connector will query data.
+- SSL certificate if you want to establish secured connections.
+
+ℹ️ User needs to set `track_commit_timestamp` to `on` using `ALTER SYSTEM SET track_commit_timestamp = on;` query to fetch last updated time.
```

Comment:
>User needs to set `track_commit_timestamp` to `on` using `ALTER SYSTEM SET track_commit_timestamp = on;` query to fetch last updated time.

I think this is best moved to the pre-requisites section.

Reply:
move to pre-requisites section